SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Utils/ErrorLogHelper.cs

```diff
@@ -159,6 +159,13 @@ public static string ReadExceptionFile(File file)
         /// </summary>
         public static void RemoveAllStoredErrorLogFiles() => Instance.InstanceRemoveAllStoredErrorLogFiles();
 
+        /// <summary>
+        /// Remove the user's name from a crash's process path.
+        /// </summary>
+        /// <param name="errorString">A string containing the username.</param>
+        /// <returns>An anonymized string where the real username is replaced by "USER".</returns>
+        internal static string ObfuscateUserName(string errorString) => Instance.InstanceObfuscateUserName(errorString);
```

Comment:
Why static + instance are required for such simple helper function?

Reply:
Because other functions in this class were implement same. Single style support.

Reply:
It was not "just because". You do not use any instance fields, so it does not make any sense in this case.